### ISSUE TITLE
Lumberjacks not dropping of Logs in storage

### ISSUE THREAD

**Issue by Darelk0** [TangoTek/TekTopia-Community#331]
**Describe the bug**
Lumberjacks are not dropping off resources at storage

**Expected behavior**
Drop items off at storage

**Screenshots**
![2019-06-07_02 43 14](https://user-images.githubusercontent.com/51474789/59089612-3711cf80-88d0-11e9-9899-5883e7f82611.png)
![2019-06-07_02 43 25](https://user-images.githubusercontent.com/51474789/59089613-37aa6600-88d0-11e9-9174-9250aa7c5486.png)
![2019-06-07_02 43 34](https://user-images.githubusercontent.com/51474789/59089614-37aa6600-88d0-11e9-918c-726a3e3782e5.png)
![2019-06-07_02 43 38](https://user-images.githubusercontent.com/51474789/59089615-37aa6600-88d0-11e9-95ee-48013c94abf8.png)
![2019-06-07_02 43 48](https://user-images.githubusercontent.com/51474789/59089616-37aa6600-88d0-11e9-87ec-f28b6a42dc34.png)

** Server/Location **
What server were you playing on? Main(Survival) or Dev (Creative)? What if Survival 2
What are the world coordinates of your town hall marker? Darloc village 3461 -8684

**Comment by JohnyDL** [comment]
Yeah I think this is related to the fix for not dropping off saplings that happened between 1.0.24 and 1.0.25, for now you're gonna have to keep an eye on them see if they hit 4 stacks of a log and then start dropping them or something like that, you might also need to start keeping them alive with potions, good luck
